### PR TITLE
Vf Adjustments

### PR DESCRIPTION
**Pull request by DXCanas**
Modals are implemented. Styling them, particularly in mobile, is going to get a little rough. This is what they look like so far. Did uber-basic styling so that you all can get an idea for what they look like.

- [x] Terms of Service Modal
- [x] Password Change Modal
- [ ]  Explicit verbage for these modals

### Desktop
image
image
image
image

### Mobile
image
image

### CODE REVIEW

**Comment by khangmach** [comment]
looks bueno, but can we make the (XXX) XXX-XXX be the same font size as the other text within that sentence. Also ToS modal on tablet can be less wide.

**Comment by DXCanas** [comment]
@khangmach styling the modals is kinda complicated, but I'll look into it.

**Comment by DXCanas** [comment]
@khangmach New mocks of modal opened

image
image

File: kolibri_instant_schools_plugin/assets/src/vue/sign-up-page/index.vue [hunk 1]
```diff
@@ -196,6 +202,17 @@
 </script>
 
 
+<style lang="stylus">
+
+  #signup-page
+    .ui-modal__
+      &container
+          max-height: 90vh
```
Comment (DXCanas) [inline kolibri_instant_schools_plugin/assets/src/vue/sign-up-page/index.vue]: using vh and vw because that's what keen uses

**Comment by indirectlylit** [comment]
looks good.

see bug fixes:
* 08224d50d9a1a7a0e6504fe3e56e46966a14aee1
* 39096045a990a35668fc872db1d6f193004c98c6

**Pull request merged**